### PR TITLE
CHORE: Add oldpreview/oldpush and warnings

### PR DESCRIPTION
**Pull request by tlimoncelli**

### CODE REVIEW

**Comment by tlimoncelli** [comment]
```
$ dnscontrol preview --domains x
[WARNING: This command will receive a major upgrade in the next release (v4.15). Try the new behavior by using ppreview/ppush. Please test and report any bugs ASAP. The older code will be available as oldpreview/oldpush for a limited number of releases.]

$ dnscontrol push --domains x
[WARNING: This command will receive a major upgrade in the next release (v4.15). Try the new behavior by using ppreview/ppush. Please test and report any bugs ASAP. The older code will be available as oldpreview/oldpush for a limited number of releases.]

$ dnscontrol oldpreview --domains x
[WARNING: oldpreview/preview are temporary commands and may go away in a future release without warning. They are not subject to SemVer. Please upgrade to ppreview/ppush instead.]

$ dnscontrol oldpush --domains x
[WARNING: oldpreview/preview are temporary commands and may go away in a future release without warning. They are not subject to SemVer. Please upgrade to ppreview/ppush instead.]
```

File: commands/previewPush.go [hunk 1]
```diff
@@ -129,13 +153,30 @@ func (args *PushArgs) flags() []cli.Flag {
 	return flags
 }
 
+const pwarn = "WARNING: This command will receive a major upgrade in the next release (v4.15). Try the new behavior by using ppreview/ppush. Please test and report any bugs ASAP. The older code will be available as oldpreview/oldpush for a limited number of releases."
+const owarn = "WARNING: oldpreview/preview are temporary commands and may go away in a future release without warning. They are not subject to SemVer. Please upgrade to ppreview/ppush instead."
```
Comment (cafferata) [inline commands/previewPush.go:157]: Shouldn't this be `oldpush`?

```suggestion
const owarn = "WARNING: oldpreview/oldpush are temporary commands and may go away in a future release without warning. They are not subject to SemVer. Please upgrade to ppreview/ppush instead."
```


```diff
$ dnscontrol oldpush
-[WARNING: oldpreview/preview are temporary commands and may go away in a future release without warning. They are not subject to SemVer. Please upgrade to ppreview/ppush instead.]
+[WARNING: oldpreview/oldpush are temporary commands and may go away in a future release without warning. They are not subject to SemVer. Please upgrade to ppreview/ppush instead.]
```

Comment (cafferata) [inline commands/previewPush.go:157]: Naming things and documentation is difficult, so be extra careful witxh choosing the right terms. Do you mean by 'Please upgrade to ppreview/ppush instead' that everyone should change their processes/code to `ppreview` and `ppush`? I would expect a swap from `preview` (serial) to `preview` (concurrent) in upcoming release(s) where our processes/code remains unchanged. The other hint about testing with `ppreview`/`ppush` certainly seems useful to me.

File: commands/previewPush.go [hunk 2]
```diff
@@ -129,13 +153,30 @@ func (args *PushArgs) flags() []cli.Flag {
...
 	return run(args, false, false, printer.DefaultPrinter, nil)
 }
 
 // Push implements the push subcommand.
 func Push(args PushArgs) error {
+	printer.Println(pwarn)
+	return run(args.PreviewArgs, true, args.Interactive, printer.DefaultPrinter, &args.Report)
+}
+
+// OldPush implements the push subcommand.
```
Comment (cafferata) [inline commands/previewPush.go:177]:
```suggestion
// OldPush implements the oldpush subcommand.
```

https://github.com/StackExchange/dnscontrol/blob/45b380cbafddb8eb9044c78cc0684769eabbd6df/commands/previewPush.go#L124-L131

**Pull request closed without merging**